### PR TITLE
Add a file for Kotor, Montenegro

### PR DESCRIPTION
**Pull request by NatashaSteblay**
Whats the purpose of this pull request - add a hometown file for Kotor
When do we think it will be merged

Some ? that I want you to all help me answer:
1. If you have been in Orlando, what did you do?  What did you think of it?
1. If you have kids, what kind of things they like to do at night?
1. What kind of food do you like?

### CODE REVIEW

**Review by mm-sfg — COMMENTED** [review]
Looks like cool places to go

**Comment by mm-sfg** [comment]
I've never been here @NatashaSteblay , but I'd like to go.

Also never been to Orlando.

File: NatashaSteblay-kotor.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#Kotor, Montenegro
+
+###Restaurants
+Pizza place
+
```
Comment (NatashaSteblay) [inline NatashaSteblay-kotor.md:6]: trying comment for code

File: NatashaSteblay-kotor.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#Kotor, Montenegro
+
```
Comment (mm-sfg) [inline NatashaSteblay-kotor.md:3]: This is a line comment. More interesting home town than mine.

Comment (mm-sfg) [reply NatashaSteblay-kotor.md:3]: @NatashaSteblay and one more line comment mentioning you.

**Comment by NatashaSteblay** [comment]
This pull Request resolves #6

**Review by mm-sfg — APPROVED** [review]
This looks good to me ready to :shipit:

**Pull request merged**